### ISSUE TITLE
249: Find textures used from UV editor for all OBJs

### ISSUE THREAD

**Issue by tngreene** [X-Plane/XPlane2Blender#445]
We must find the Textures from the UV editor for all OBJs and save those in the Manual Entry Texture Path boxes. We can't forget a single scene might have multiple Root Objects in it.

We aren't going to waste time with Material's Texture Slot pattern because as it turns out most people don't use that (See #449), Blender 2.8 doesn't support it, and it would actually conflict with anything the UV editor is showing.

- [x] Write the code
- [x] Write the unit test

**Comment by tngreene** [comment]
If the user opts for this, we'll also have to
- [ ] Decide what should happen with their UV image editor image. Should it be turned off? Otherwise, what was the point for the 3D Viewer's sake?

**Comment by tngreene** [comment]
I've discovered something bad: A 2.79 file loader bug.

Some image datablock names appear to be broken. For the C90 I had to erase C90B_fuselage and retype it in. Then it worked.

**Comment by tngreene** [comment]
Closed with 7ffed0cad5721cdfbb5a6!

**Comment by tngreene** [comment]
Re-opened.

```
We aren't covering 
TEXTURE		738gear.dds
TEXTURE_NORMAL 738gear_NML.png
GLOBAL_specular 1.0
TEXTURE_LIT 738gear_LIT.dds
```

**Comment by tngreene** [comment]
Closed with ba87583f4!
